src/main/java/io/zohoinventory/ZohoInventorySdk.java

```diff
@@ -0,0 +1,101 @@
+package io.zohoinventory;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.zohoinventory.models.authenticationapi.AccessCredentials;
+import io.zohoinventory.models.authenticationapi.AccessTokenResponse;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.SneakyThrows;
+import org.apache.http.HttpHeaders;
+import org.apache.http.client.utils.URIBuilder;
+
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.LocalDateTime;
+import java.util.HashMap;
+import java.util.concurrent.CompletableFuture;
+
+import static io.zohoinventory.constantcode.ConstantCodes.*;
+
+@AllArgsConstructor
+@Builder(toBuilder = true)
+public class ZohoInventorySdk {
+
+    protected HttpClient client;
+    protected AccessCredentials accessCredential;
```

Comment:
Add declaration in ascending order or length.

Reply:
made declaration in ascending order